### PR TITLE
Container -> Screen handler (second attempt)

### LINKED ISSUE #61: Container -> ScreenHandler
**Names of the classes**
container becomes screenhandler

**Additional Context**
yarn parity

### PR DESCRIPTION
**Pull request by BoogieMonster1O1**
Upstream fabric calls it screen handler. 
Pretty much sums it up. 

Closes #61 
Supersedes #64

### CODE REVIEW

File: mappings/net/minecraft/client/gui/screen/ingame/BrewingScreen.mapping [hunk 1]
```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_790 net/minecraft/client/gui/screen/ingame/BrewingStandScreen
```
Comment (hYdos) [inline mappings/net/minecraft/client/gui/screen/ingame/BrewingScreen.mapping:1]: bad. old name is more yarn here

File: mappings/net/minecraft/client/gui/screen/ingame/CraftingScreen.mapping [hunk 2]
```diff
@@ -1,4 +1,4 @@
-CLASS net/minecraft/class_793 net/minecraft/client/gui/screen/ingame/CraftingTableScreen
+CLASS net/minecraft/class_793 net/minecraft/client/gui/screen/ingame/CraftingScreen
```
Comment (hYdos) [inline mappings/net/minecraft/client/gui/screen/ingame/CraftingScreen.mapping:1]: bad. less yarn is not good. revert this

File: mappings/net/minecraft/server/network/ServerPlayerEntity.mapping [hunk 3]
```diff
@@ -33,13 +34,15 @@ CLASS net/minecraft/class_1637 net/minecraft/server/network/ServerPlayerEntity
 		ARG 1 distance
 	METHOD method_6066 updateBlockEntity (Lnet/minecraft/class_348;)V
 		ARG 1 blockEntity
-	METHOD method_6069 openContainer (Lnet/minecraft/class_1983;)V
+	METHOD method_6069 registerScreenHandler (Lnet/minecraft/class_1983;)V
```
Comment (shedaniel) [inline mappings/net/minecraft/server/network/ServerPlayerEntity.mapping:37]: not open?

File: mappings/net/minecraft/entity/ai/AIGoals.mapping [hunk 4]
```diff
@@ -1,17 +1,17 @@
-CLASS net/minecraft/class_1790 net/minecraft/entity/ai/AIGoals
+CLASS net/minecraft/class_1790 net/minecraft/entity/ai/AiGoals
 	FIELD field_7650 LOGGER Lorg/apache/logging/log4j/Logger;
 	FIELD field_7652 goalEntries Ljava/util/List;
 	FIELD field_7653 profiler Lnet/minecraft/class_1705;
 	METHOD <init> (Lnet/minecraft/class_1705;)V
 		ARG 1 profiler
-	METHOD method_7371 addGoal (ILnet/minecraft/class_1789;)V
```
Comment (hYdos) [inline mappings/net/minecraft/entity/ai/AIGoals.mapping:7]: addGoal is a better name

File: mappings/net/minecraft/server/network/ServerPlayerEntity.mapping (hunk 3 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/server/network/ServerPlayerEntity.mapping:37]: yep. 
it actually sends packets to the client with screen handler info

File: mappings/net/minecraft/screen/NamedScreenHandlerFactory.mapping [hunk 5]
```diff
@@ -0,0 +1,5 @@
+CLASS net/minecraft/class_1723 net/minecraft/screen/NamedScreenHandlerFactory
+	METHOD method_6761 createMenu (Lnet/minecraft/class_1962;Lnet/minecraft/class_1963;)Lnet/minecraft/class_1983;
```
Comment (hYdos) [inline mappings/net/minecraft/screen/NamedScreenHandlerFactory.mapping:2]: createScreen ?

File: mappings/net/minecraft/server/network/ServerPlayerEntity.mapping (hunk 3 above)
Comment (shedaniel) [reply mappings/net/minecraft/server/network/ServerPlayerEntity.mapping:37]: it doesn't register it right

File: mappings/net/minecraft/util/PacketByteBuf.mapping [hunk 6]
```diff
@@ -160,6 +160,8 @@ CLASS net/minecraft/class_1435 net/minecraft/util/PacketByteBuf
 	METHOD setShort (II)Lio/netty/buffer/ByteBuf;
 		ARG 1 i
 		ARG 2 j
-	METHOD writeBytes (Lio/netty/buffer/ByteBuf;)Lio/netty/buffer/ByteBuf;
```
Comment (hYdos) [inline mappings/net/minecraft/util/PacketByteBuf.mapping:163]: its a byte buf. not a byte. the correct name is writeBytes

**Review by hYdos — CHANGES_REQUESTED** [review]
this is quite an important update. im going to add liach to the requested reviewers. im not 100% sure on some of these names

File: mappings/net/minecraft/server/network/ServerPlayerEntity.mapping (hunk 3 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/server/network/ServerPlayerEntity.mapping:37]: wait it doesnt

File: mappings/net/minecraft/screen/NamedScreenHandlerFactory.mapping (hunk 5 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/screen/NamedScreenHandlerFactory.mapping:2]: upstream calls it `createMenu` even if it's not supposed to be called that

File: mappings/net/minecraft/util/PacketByteBuf.mapping (hunk 6 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/util/PacketByteBuf.mapping:163]: i
did
not
do
that

packetbytebuf
has
all
methods
deobf

File: mappings/net/minecraft/server/network/ServerPlayerEntity.mapping (hunk 3 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/server/network/ServerPlayerEntity.mapping:37]: fixed

Comment (BoogieMonster1O1) [reply mappings/net/minecraft/server/network/ServerPlayerEntity.mapping:37]: `updateScreenHandler`

File: mappings/net/minecraft/client/gui/screen/ingame/CraftingScreen.mapping (hunk 2 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/client/gui/screen/ingame/CraftingScreen.mapping:1]: image

File: mappings/net/minecraft/client/gui/screen/ingame/BrewingScreen.mapping (hunk 1 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/client/gui/screen/ingame/BrewingScreen.mapping:1]: image

File: mappings/net/minecraft/entity/ai/AIGoals.mapping (hunk 4 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/entity/ai/AIGoals.mapping:7]: image
the class only adds one thing

**Comment by BoogieMonster1O1** [comment]
#64, which does exactly this wasn't scrutinized this much

File: mappings/net/minecraft/entity/ai/AIGoals.mapping (hunk 4 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/entity/ai/AIGoals.mapping:7]: upsream calls it `add`

**Comment by BoogieMonster1O1** [comment]
> this is quite an important update. im going to add liach to the requested reviewers. im not 100% sure on some of these names

@hYdos does this mean this won't get approved and/or merged until liach reviews?

File: mappings/net/minecraft/util/PacketByteBuf.mapping (hunk 6 above)
Comment (hYdos) [reply mappings/net/minecraft/util/PacketByteBuf.mapping:163]: you cant say you didnt do that. its literately a change in the file. from YOUR pr

**Comment by hYdos** [comment]
> > this is quite an important update. im going to add liach to the requested reviewers. im not 100% sure on some of these names
> 
> @hYdos does this mean this won't get approved and/or merged until liach reviews?

yes, sadly

**Comment by hYdos** [comment]
also can you please keep your mappings on topic of the pr? i dont want to have to review an extra 50 files because you wanted to do more

File: mappings/net/minecraft/util/PacketByteBuf.mapping (hunk 6 above)
Comment (BoogieMonster1O1) [reply mappings/net/minecraft/util/PacketByteBuf.mapping:163]: It doesn’t matter
All methods are deobfuscated

**Comment by BoogieMonster1O1** [comment]
> also can you please keep your mappings on topic of the pr? i dont want to have to review an extra 50 files because you wanted to do more

sure

**Review by hYdos — CHANGES_REQUESTED** [review]
fix class names.

**Comment by BoogieMonster1O1** [comment]
> fix class names.

no

**Pull request merged**